app/root.css

```diff
@@ -840,3 +844,30 @@ button {
 }
 
 /* end mobile */
+.video-thumbnail-wrapper {
+  position: relative;
```

Comment:
maybe just add a generic `relative` class that can be applied to arbitrary elements? I'd put these specific classes in a separate css file next to the component that they apply to